### DISCUSSION TITLE
Bun v0.8.1

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#4313] [Announcements]
To install:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade:
```bash
bun upgrade
```

**[Read Bun v0.8.1's release notes on Bun's blog](https://bun.sh/blog/bun-v0.8.1)**

## What's Changed
* Add Debugger docs and a couple guides by @colinhacks in https://github.com/oven-sh/bun/pull/4281
* 🚰  drain the microtasks more by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/4282
* Fix performance regression in reading from the request body by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/4291
* Update bun-polyfills & bun-wasm by @jhmaster2000 in https://github.com/oven-sh/bun/pull/4246
* update zig to 0.11.0 by @CGQAQ in https://github.com/oven-sh/bun/pull/4233
* Don't inline require/import errors at runtime by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/4306
* Make the server not crash if an error occurs in the dev build. by @Hanaasagi in https://github.com/oven-sh/bun/pull/4300
* Fix(node:http): fix URL formatting when using a proxy. by @Hanaasagi in https://github.com/oven-sh/bun/pull/4297
* [install] fix crash when installing package that uses loose semver pre-release by @alexlamsl in https://github.com/oven-sh/bun/pull/4302
* [install] fix stale life-cycle scripts from lockfile by @alexlamsl in https://github.com/oven-sh/bun/pull/4307
* Report extra memory more by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/4289
* Remove conflict option in tsconfig-for-init.json by @codehz in https://github.com/oven-sh/bun/pull/4284
* Add guides for test runner by @colinhacks in https://github.com/oven-sh/bun/pull/4308
* Listen on a unix domain socket with Bun.serve() by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/4311
* Improved support for `debug-adapter-protocol` by @Electroid in https://github.com/oven-sh/bun/pull/4186

## New Contributors
* @CGQAQ made their first contribution in https://github.com/oven-sh/bun/pull/4233

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v0.8.0...bun-v0.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v0.8.1'>Bun v0.8.1</a>.</em>

**Comment by Bhanusingh99** [comment]
Subject: Collaboration Opportunity: Promote Your Bun Products to Our Engaged Tech Community!

Dear [Bun Corporation's Name],

I hope this message finds you well. My name is [Bhanu singh], and I represent a dynamic duo of tech enthusiasts and micro Twitter influencers. We are excited to reach out to Bun Corporation with a unique collaboration proposal that we believe can benefit both of our brands.

Firstly, we wanted to applaud your incredible Bun products, which we've personally found to be incredibly useful and innovative. Our tech-savvy audience on Twitter is always on the lookout for practical solutions that can improve their daily lives, and we believe Bun products align perfectly with their needs.

Here's a brief overview of what we can offer:

1. **Engaged Twitter Community:** As micro-influencers in the tech niche, we have cultivated a loyal and active following within the Twitter tech community. On average, our tweets garner approximately 30,000 views per post, and this number is steadily growing.

2. **Specialized Content Creation:** We are eager to create two engaging and informative posts specifically dedicated to Bun products. These posts will highlight the unique features and benefits of using your products, ensuring they reach a highly receptive audience.

3. **Impressive Reach:** Our most recent tweet about bun-related content garnered an astonishing 15,000 views within just 6 hours, showcasing our ability to capture our audience's attention effectively.

4. **Affordable Pricing:** We understand the value of effective marketing and would like to offer our services at a competitive rate of $250 for both posts.

We believe that by partnering with us, Bun Corporation can tap into a highly targeted and enthusiastic customer base, while we can continue to provide our followers with valuable and innovative tech solutions.

If you're interested in discussing this collaboration further or have any questions, please feel free to reach out to us at [Your Email Address]. We would be delighted to explore how we can work together to elevate the visibility of Bun products in the tech community.

Thank you for considering our proposal, and we look forward to the possibility of collaborating with Bun Corporation.

Warm regards,

[Bhanu singh ]
our Twitter Handles
@Bhanu_here
@maybeshalinii
[bhanusngh12345@gmail.com]

sorry for writing here but I was not able to find Bun official email so i am writing this text over here
